src/main/scala/io/iohk/ethereum/vm/ProgramContext.scala

```diff
@@ -14,9 +15,14 @@ object ProgramContext {
 
     import stx.tx
 
+    // YP eq (91)
+    val programInput =
```

Comment:
Minor: for constistency I would rename it to `inputData`. That's how it's referred to everywhere else.